libraries/WiFi/src/WiFiGeneric.cpp

```diff
@@ -1473,6 +1492,37 @@ int WiFiGenericClass::hostByName(const char* aHostname, IPAddress& aResult)
     return (uint32_t)aResult != 0;
 }
 
+/**
+ * Resolve the given hostname to an IP6 address.
+ * @param aHostname     Name to be resolved
+ * @param aResult       IPv6Address structure to store the returned IP address
```

Comment:
Return could be IPv6 or IPv4, so change comment:

* @param aResult       IP address structure to store the returned IP address, either IPv6 if available, otherwise IPv4